### PR TITLE
Update path to artifacts after successful build.

### PR DESCRIPTION
**Pull request by MalcolmAnderson**
It took me a little while to find my artifacts, I'm assuming I'm not the only new person trying out the mod before it goes to release.

### CODE REVIEW

**Comment by FrancisDaPugtato** [comment]
Oh yes, I did have this problem too. correct one is BC/Build/Libs, right?

**Comment by FrancisDaPugtato** [comment]
But you,mean change th README file, right? Also the website shows the latest test/dev build so there is no reason to do that...

**Comment by FrancisDaPugtato** [comment]
Do it if you want it thought :D

**Comment by MalcolmAnderson** [comment]
To whom it may concern, I'm not clear what steps need to be taken for this pull request to go through.  I see that the jenkins pr-merge is failing but at the same time "This branch has no conflicts with the base branch"
What steps do I need to take?

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ PR implementing new features or changing large portions of code are helpful. But
  * `gradlew setupDecompWorkspace` to setup a complete development environment.
  * With `Gradle` installed: use `gradle` instead of `gradlew`
  * On Windows: use `gradlew.bat` instead of `gradlew`
-1. The compiles and obfuscated module jars will be in 'baseDir/BuildCraft/modules'
+1. The compiles and obfuscated module jars will be in 'baseDir/BuildCraft/build/libs/&lt;build number>/modules'
```
Comment (AEnterprise) [inline README.md]: any reason you use the html one first and then the normal one? looks a bit weird

**Comment by AlexIIL** [comment]
Sorry for not merging this before, this is useful.

**Comment by MalcolmAnderson** [comment]
Thanks all, this was my first commit to an open source project.

**Pull request merged**